src/gui/rss/automatedrssdownloader.ui

```diff
@@ -107,6 +107,17 @@
            </widget>
           </item>
           <item>
+           <widget class="QCheckBox" name="checkSmart">
+            <property name="toolTip">
+             <string>Smart Episode Filter will check the episode number to prevent downloading of duplicates.
+Supports the formats: S01E01, 1x1, 2017.01.01 and 01.01.2017 (Date formats also support - as a separator)</string>
```

Comment:
Hmm, that's odd. I entered \n into Qt Creator, and I that's what it wrote to the .ui file. I tried forcing \n in the file and it doesn't work, the tooltip shows a literal '\n'. I tried using <br> instead, but that starts wrapping the text on multiple lines. It seems you either have to use an actual newline in the .ui, or some weird <nobr> nonsense.